### ISSUE TITLE
[Feature Request] Can we get a spinning circle during a metadata refresh

### ISSUE THREAD

**Issue by PlexMovieFan** [plexinc/WebClient.bundle#55]
When updating a library you get the spinning circle so you know not to mess in that library until it is done updating.  I would like to see the same or similar when performing a refresh.  Currently, the only way to know it is done is to monitor the activity page and wait for the finish message.  A nice visual indicator would be super helpful.

**Comment by ericmatthys** [comment]
I agree. Especially when downloading / loading metadata can take awhile, the inactivity could be confusing.
